### ISSUE TITLE
AI Docs Assistance 

### ISSUE THREAD

**Issue by fire17** [sveltejs/kit#10347]
### Describe the problem

Hi there, I would love to get into sveltekit and svelte, today most of my education comes from ChatGPT but I'm always feeling like it's not fully caught up with modern sveltekit. 

### Describe the proposed solution

I'd love it if you added an ai chat to the docs,
there I can ask it to help me write my python-svelte hybrid app and actually get modern recommended practices and code.
You can have a toggle option in the chat to switch between [Svelte Docs only] or [Docs + General] so I can either research something that is svelte specific or ask for hybrid/generic prompts like my example to combine python and svelte, and how it should be done correctly for my usecase.

### Alternatives considered

This can sort of be done now with Quivr or others if I start giving it all the docs' urls
But I really recommend you check Quivr out, it's foss, and easily integratable with tokens and an API and seems a perfect base for this case

### Importance

i cannot use SvelteKit without it

### Additional Information

As I use ai models alot , I like my backend in python
The only downside in python for me is the web integration. Most python-web frameworks are very rigid. I'd love to get svelte into the mix.
I have created a microservice framework with realtime coms, db integration, requests, events and pubsub, and now I want to integrate web connectivity. 
I'd love to make a 2 way binded, realtime element in svelte that is connected to my  python code, where any changes in the back send an event to immediately render on the page, and any changes in the page go directly into my python service 
Please let me know if you have good material for a Realtime python-svelte stack. I've been looking and it's really hard to find.
A Svelte-aware AI could really help me at this point
Thanks a lot! & All the best !

**Comment by fire17** [comment]
Btw the current best materials I found so far are:
YouTube - [FastAPI+Sveltekit](https://youtu.be/qXtcidB6qhM)

**Comment by Conduitry** [comment]
No. We're not interested in developing (and paying for) something that we really have no idea whether it will actually give correct or useful answers. Projects that take great care in their documentation do so for a reason, and it's so that people can read it, not so that AIs can ingest it and then return a distorted version of that documentation.

**Comment by fire17** [comment]
> No. We're not interested in developing (and paying for) something that we really have no idea whether it will actually give correct or useful answers. Projects that take great care in their documentation do so for a reason, and it's so that people can read it, not so that AIs can ingest it and then return a distorted version of that documentation.

Wow , why so blatant ? Why you close this so quickly? Is this your personal opinion? @Conduitry 

I don't think you realize how much value this can bring to people, and yes you can disclaim it by saying that it's experimental

Instead of closing this you should consider getting other's opinions

I think you should put real effort to making sure it's good and puting it on the front page

I think you should take a moment to care about me spending hours going through research and trying to build something with svelte and  consider that this is a supercharger and will greatly help people like me 
It's insane to me that chatgpt is not up-to-date with latest svelte and sveltekit docs (which change all the time) you can actually do something about it.

You can use free local models , or let people input their api keys to use if you don't want to pay
But actually, @Rich-Harris should break a deal with vercel to pay for the gpt3.5/4 calls to make sure the answers are highest quality

Honestly don't sleep on this.
This + a foss wix-style svelte Visual website builder should be on your top priority

**Comment by PuruVJ** [comment]
Hi! Thanks for your suggestion.

**tl;dr**: GPT lies, can't have lies in official docs

The thing is, the AI engines like GPT 3.5 and even 4 are actually quite inaccurate. These are made to write out text like humans do, but they are not fully accurate about what they're writing. This poses the challenge of showing inaccurate information in the official docs. 

MDN, for instance did this experiment recently. their version gives off wrong information in some cases. Imagine if you go to MDN and it tells you wrong information about what css `grid: span 1 / 1` does. They themselves decided to shut it down, while they research ways to make it more accurate. Or maybe we won't see MDN Ai again for a year or two while GPT 5 or 6 or a newer and better engine comes out. Only time will tell.

So learning from their mistake, we feel it's not right to have such a thing in official docs. I believe @ghostdevv is looking into ways of integrating GPT4 in our discord server, so you'll still be able to use GPT trained on svelte docs within our discord.

Hope that answers your questions!

**Comment by ghostdevv** [comment]
> So learning from their mistake, we feel it's not right to have such a thing in official docs. I believe @ghostdevv is looking into ways of integrating GPT4 in our discord server, so you'll still be able to use GPT trained on svelte docs within our discord.

Yep! Soon :tm:

**Comment by fire17** [comment]
Very happy to hear :) @ghostdevv 

Anyways I hope you reconsider, if some already  train or setup an LLM with up-to-date sveltekit knowledge and good feedback is coming, please make a port on the docs' site

I'm guessing soon we'll get tools that can just permissionlessly injest any large docs site , like svelte's, and we'll just be able to do that. But still it will probably be a slow rollout to early adopters. And I would much rather do this in your svelte space. See the orange.

(Or wait till react and Vue do it and play catch up)

Anyways for now I will just do like the good old days, post aomething in the discussion or issues, share my progress, get natural language human feedback & suggestions , and hopefully solve my issue.

Thanks a lot and all the best !
